docs/integrations/sources/freshdesk.md

```diff
@@ -27,7 +27,7 @@ If there are more endpoints you'd like Airbyte to support, please [create an iss
 | Feature | Supported? |
 | :--- | :--- |
 | Full Refresh Sync | Yes |
-| Incremental Sync | Coming soon |
+| Incremental Sync | Yes (except Conversations) |
 | Replicate Incremental Deletes | Coming soon |
```

Comment:
The API has no concept of this, right? so we shouldn't really mention it in the docs
```suggestion
```